extensions/terminal-suggest/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terminal-suggest",
+  "publisher": "vscode",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "1.0.1",
+  "private": true,
+  "license": "MIT",
+  "engines": {
+    "vscode": "0.10.x"
+  },
+  "icon": "images/npm_icon.png",
+  "categories": [
+    "Other"
+  ],
+  "enabledApiProposals": [
+    "terminalCompletionProvider"
+  ],
+  "scripts": {
+    "compile": "npx gulp compile-extension:npm",
+    "watch": "npx gulp watch-extension:npm"
+  },
+  "dependencies": {
+    "@fig/autocomplete-generators": "^2.4.0",
+    "find-up": "^5.0.0",
+    "find-yarn-workspace-root": "^2.0.0",
+    "jsonc-parser": "^3.2.0",
+    "minimatch": "^5.1.6",
+    "request-light": "^0.7.0",
+    "vscode-uri": "^3.0.8",
+    "which": "^4.0.0",
+    "which-pm": "^2.1.1"
+  },
+  "devDependencies": {
+    "@types/minimatch": "^5.1.2",
+    "@types/node": "20.x",
+    "@types/which": "^3.0.0"
+  },
+  "main": "./out/terminalSuggestMain",
+  "activationEvents": [
+    "*"
+  ],
```

Comment:
I think we need an associated activation event for the providers?

Reply:
should we add something like this 🤔 

![Screenshot 2024-11-13 at 12 04 52 PM](https://github.com/user-attachments/assets/add75040-3852-4f1d-a380-14d494dc222c)

Reply:
hm, or probably better to have the event specific to the suggest addon?